src/libexpr/nixexpr.hh

```diff
@@ -20,18 +20,24 @@ MakeError(Abort, EvalError);
 MakeError(TypeError, EvalError);
 MakeError(UndefinedVarError, Error);
 MakeError(MissingArgumentError, EvalError);
-MakeError(RestrictedPathError, Error);
 
 /* Position objects. */
-
 struct Pos
 {
-    std::string file;
-    FileOrigin origin;
     uint32_t line;
     uint32_t column;
 
+    struct no_pos_tag {};
```

Comment:
The others are clear with a bit of thinking, but under what circumstances would this one end up being used?

Reply:
I got rid of it, it was only used internally in `PosTable::operator []`.